### PR TITLE
allow administrator to manage organisations and domain names

### PR DESCRIPTION
**Pull request by RignonNoel**
| Q                                          | R
| ------------------------------------------ | -------------------------------------------
| Type of contribution ?                     | Features
| Tickets (_issues_) concerned               | [39](https://genielibre.com/issues/39)

---

##### What have you changed ?
 - Although the issue speaks only of university I allow user to add organizations. If a day other types of organization appear we can then add a type field.
 - Allow administrator to manage domains name of `organizations`
 - Add a check for I18N in PR template (to follow this first review of @francisjeanneau )

##### How did you change it ?
 - Add a new model `Organization`.
 - Add a new model `Domain`.
 - Add a custom admin to add domain name in the organization admin page.

##### Is there a special consideration?

I didn't add some test on these model since we just have a `__str()__` method and it's not use directly in the API. I thought it was better to focus on the functional aspect and not disperse it on this kind of test.

Verification :
 -  [x] My branch name respect the standard defined in `CONTRIBUTING.md`
 -  [x] All my commits respect the standard defined in `CONTRIBUTING.md`
 -  [x] This Pull-Request fully meets the requirements defined in the issue
 -  [ ] I added or modified the attached tests
  - See special consideration for explanation
 -  [ ] I added or modified the documentation
  - No documentation on this part of the system for the moment

### CODE REVIEW

File: blitz_api/admin.py [hunk 1]
```diff
@@ -1,5 +1,20 @@
 from django.contrib import admin
 from django.contrib.auth.admin import UserAdmin
-from .models import User
+from .models import User, Organization, Domain
+
+
+class DomainInline(admin.StackedInline):
+    model = Domain
+    can_delete = True
+    verbose_name_plural = 'Domains'
```
Comment (francisjeanneau) [inline blitz_api/admin.py]: ugettext_lazy('Domains')

File: blitz_api/models.py [hunk 2]
```diff
@@ -35,3 +35,36 @@ def expire(self):
         """Expires a token by setting its expiration date to now."""
         self.expires = timezone.now()
         self.save()
+
+
+class Organization(models.Model):
+    """Represent an existing organization such an university in the world"""
+
+    name = models.CharField(
+        verbose_name="Name",
```
Comment (francisjeanneau) [inline blitz_api/models.py]: ugettext_lazy()

File: blitz_api/models.py [hunk 3]
```diff
@@ -35,3 +35,36 @@ def expire(self):
...
+
+
+class Domain(models.Model):
+    """An internet domain name like fsf.org"""
+
+    # Full domain name may not exceed 253 characters in its textual
+    # representation :
+    # https://en.wikipedia.org/wiki/Domain_Name_System#Domain_name_syntax
+    name = models.CharField(
+        verbose_name="Name",
```
Comment (francisjeanneau) [inline blitz_api/models.py]: ugettext_lazy()

**Review by francisjeanneau — CHANGES_REQUESTED** [review]
Il va falloir ajouter la classe Meta() au modèles pour internationaliser `verbose_name`.

Je sais pas si c'est la meilleure méthode, mais c'est ce que j'ai trouvé rapidement:
https://stackoverflow.com/questions/2938692/django-internationalization-for-admin-pages-translate-model-name-and-attribute

**Comment by RignonNoel** [comment]
C'est effectivement comme cela que je fais d'habitude. 

 - I18N ajoute
 - J'ai aussi ajoute un check dans le PR template pour l'I18N

File: blitz_api/models.py [hunk 4]
```diff
@@ -35,3 +42,44 @@ def expire(self):
         """Expires a token by setting its expiration date to now."""
         self.expires = timezone.now()
         self.save()
+
+
+class Organization(models.Model):
+    """Represent an existing organization such an university in the world"""
```
Comment (francisjeanneau) [inline blitz_api/models.py]: Je remplacerais par:
`Represents an existing organization such as an university`

Comment (RignonNoel) [reply blitz_api/models.py]: :+1:

**Review by francisjeanneau — APPROVED** [review]

**Pull request merged**